docs/C_API.md

```diff
@@ -29,3 +29,12 @@
 The example below shows a sample run using the SqueezeNet model from ONNX model zoo, including dynamically reading model inputs, outputs, shape and type information, as well as running a sample vector and fetching the resulting class probabilities for inspection.
 
 * [../csharp/test/Microsoft.ML.OnnxRuntime.EndToEndTests.Capi/C_Api_Sample.cpp](../csharp/test/Microsoft.ML.OnnxRuntime.EndToEndTests.Capi/C_Api_Sample.cpp)
+
+# Telemetry
+This project collects usage data and sends it to Microsoft to help improve our products and services. Note however that no data collection is performed by default when using your private builds
+
+Telemetry is turned on by default when using the Official Builds.  This is implemented via 'Platform Telemetry' per vendor platform providers (see telemetry.h).
```

Comment:
Turned "off" by default. Also, is it better to write the documentation in one place and link to it from everywhere else?

Reply:
check out my comments on the other doc on why i like have the "spec" be ON by default, and the code will catch up later when we are ready (bug level fixes) .    as to the docs spread around a bit, i went back and forth on it :) .   pros/cons.

1. other OSS components have multiple docs.  having things like a "privacy statment" are nice.
2. i like the main README being more of a TOC and link off.  that way the root/readme doesn't just bloat and bloat with content.
3. i like the idea of drill down for details.  the top level docs have basic ideas.  and you get to go down deeper (like the c_api doc) to learn how to use the actual C api in this fashion.

what ya think?

---
In reply to: [335297643](https://github.com/microsoft/onnxruntime/pull/2109#discussion_r335297643) [](ancestors = 335297643)

Reply:
It's ok as long as we don't end up with inconsistent documentation. @faxu can probably review this and give some comments.